java-tools/OsmAndServerUtilities/src/main/java/net/osmand/server/osmgpx/DownloadOsmGPX.java

```diff
@@ -136,7 +129,7 @@ public static void main(String[] args) throws Exception {
 				switch (s[0]) {
 				case "--acitivity-type":
 					if (val.trim().length() > 0) {
-						qp.activityTypes = new TreeSet<RouteActivityType>();
```

Comment:
why it's updated ?

Reply:
Because Exception in thread "main" java.lang.ClassCastException: class net.osmand.osm.RouteActivityType cannot be cast to class java.lang.Comparable